app/elements/pw-shell.html

```diff
@@ -223,6 +251,14 @@
       _searchParams: {
         type: Object,
         observer: '_searchParamsChanged'
+      },
+
+      /**
+       * Which version (1.0 or 2.0) of the docs we are using, so that the
+       * main nav and side nav can be kept in sync across page navigations.
+       */
+      activeDocsVersion: {
+        type: String
```

Comment:
Instead of maintaining a separate "version" property and updating it whenever the URL changes, you can just use `this._sectionData.version` which is maintained by `app-route` (or make this a computed property based on that).